content/examples/databricks-cluster.Rmd

```diff
@@ -0,0 +1,128 @@
+---
+title: "Using sparklyr with Databricks"
+output:
+  html_document:
+    toc: true
+    toc_depth: 2
+    toc_float:
+      collapsed: false
+params:
+  width: 600
+---
+
+## Overview
+
+This documentation demonstrates how to use `sparklyr` with RStudio Server Pro,
+RStudio Connect, and RStudio Package Manager to work with Databricks.
+
+## Best practices
+
+- **Maintain separate installation environments** - Install RStudio Server Pro,
+  RStudio Connect, and RStudio Package Manager outside of the Databricks cluster
+  so that they are not limited to the resources or ephemeral nature of
+  Databricks clusters.
+- **Connect to Databricks remotely** - Work with Databricks as a remote compute
+  resource in RStudio Server Pro, similar to how you would connect remotely to
+  external databases, data sources, and storage systems. This can be
+  accomplished using Databricks Connect as described in the
+  [Connecting to Databricks remotely](#option-1---connecting-to-databricks-remotely)
+  section below, the
+  [RStudio Professional Drivers](https://docs.rstudio.com/pro-drivers/) for Hive
+  and Impala, or the JDBC/ODBC server in Databricks on
+  [AWS](https://docs.databricks.com/integrations/bi/jdbc-odbc-bi.html) or
+  [Azure](https://docs.microsoft.com/en-us/azure/databricks/integrations/bi/jdbc-odbc-bi).
+- **Restrict workloads to interactive analysis** - Only perform workloads
+  related to exploratory or interactive analysis with Spark, then write the
+  results to a database, file system, or cloud storage for more efficient
+  retrieval in apps, reports, and APIs.
+- **Load and query results efficiently** - Because of the nature of Spark
+  computations and the associated overhead, Shiny apps that use Spark on the
+  backend tend to have performance and runtime issues; consider reading the
+  results from a database, file system, or cloud storage instead.
+
+## Options for using RStudio Server Pro with Databricks
```

Comment:
I would call this `Using RStudio Server Pro with Databricks` and then give Option 1 and Option 2 level 3 headings. That might mess up the ToC (not sure how many nested levels it shows) but it would be consistent with how the rest of the document is outlined.

Reply:
Done in 7f2c693, thanks!